berserk/clients/challenges.py

```diff
@@ -173,3 +181,56 @@ def decline(
         path = f"/api/challenge/{challenge_id}/decline"
         payload = {"reason": reason}
         self._r.post(path, json=payload)
+
+    def cancel(self, challenge_id: str, opponent_token: str | None = None) -> None:
+        """Cancel an outgoing challenge, or abort the game if challenge was accepted but the game was not yet played.
+
+        :param challenge_id: ID of a challenge
+        :param opponent_token: if set to the challenge:write token of the opponent, allows game to be cancelled
+            even if both players have moved
+        """
+        path = f"/api/challenge/{challenge_id}/cancel"
+        params = {"opponentToken": opponent_token}
+        self._r.post(path=path, params=params)
+
+    def start_clocks(
+        self, game_id: str, token_player_1: str, token_player_2: str
+    ) -> None:
+        """Starts the clocks of a game immediately, even if a player has not yet made a move.
+
+        Requires the OAuth tokens of both players with challenge:write scope.
+
+        If the clocks have already started, the call will have no effect.
+
+        :param game_id: game ID
+        :param token_player_1: OAuth token of player 1 with challenge:write scope
+        :param token_player_2: OAuth token of player 2 with challenge:write scope
```

Comment:
The docs here kinda suggest there is a specific order to the players (e.g. maybe player1 == white) but I'm pretty sure there isn't. It just has to be one token from white and one from black, in either order. I'd maybe just add a sentence or two explaining that.